OpenCO2_Sensor.ino

```diff
@@ -586,10 +588,34 @@ void startWiFi() {
   wifiManager.setConfigPortalBlocking(false);
   wifiManager.autoConnect("OpenCO2 Sensor");  // name of broadcasted SSID  
 
+  // is this needed?
+  while(WiFi.status() != WL_CONNECTED)
+      delay(10);
+  delay(5000);
+
```

Comment:
I thought maybe the wifi isn't really ready or stable yet. But otoh, I am seeing the connection attempt in mosquitto.log, so guess this is not needed.